### ISSUE TITLE
Valentin sees the new page model in Elastic

### ISSUE THREAD

**Issue by lestrade-pbstck** [pbstck/models#36]
## Why
We need to ingest all of previously added fields in athena and es

## Context

[User sessions pitch #2](https://www.notion.so/User-Sessions-2-bf5819b425924d10a76a7fd8fec43f00)

## Demo
* Valentin goes to page table in Athena
* Valentin sees the new fields (utm fields, referrer, consent,....)
* Valentin goes to elasticsearch and sees the same fields

**Comment by lestrade-pbstck** [comment]
> Don't you need to update es-tooling project as well to be able to store the new values in ES ?

Yes, but as it's not directly related to the model project, we did not add it in this US. But we'll have to create an US for es tooling project

**Comment by vmaerten** [comment]
I would separate it into two US
- One for Athena
- One for OpenSearch
As you don't know what to put in OS AND it's better to validate in Athena before IMHO
WDYT ?

**Comment by lestrade-pbstck** [comment]
That's our bad, we misnamed the US, this one should be for Athena only :sweat_smile: Changed the title to replace Elastic for Athena. We'll create the Elastic one when we know more about the dashboards
